### PR TITLE
Liveview (0.19) & deps updated

### PR DESCRIPTION
**Pull request by nathanmalishev**
Updated mix to use Liveview 0.19 and updated other decencies as well. 

Modified the test that checks the 'Global attribute' docs as it must have changed as well. (test/heroicons_test.exs:96).

Issue - https://github.com/mveytsman/heroicons_elixir/issues/24

### CODE REVIEW

**Comment by dovadi** [comment]
Concerning the failing "generating docs" test: since phoenix_live_view 0.18.4 the function [build_attrs_docs](https://github.com/phoenixframework/phoenix_live_view/blob/3d7a3bace5ba5ff54359e7d2e21f96b5236579c0/lib/phoenix_component/declarative.ex#L815) is changed. In case of a global attribute you can't declare documentation for the attribute anymore, it just returns _"* Global attributes are accepted.\n"._

It seems inconsistent, because in the phoenix_live_view library itself in case of [a global attribute](https://github.com/phoenixframework/phoenix_live_view/blob/3d7a3bace5ba5ff54359e7d2e21f96b5236579c0/lib/phoenix_component.ex#L2134) document information is still provided

You can see this in the documentation of Phoenix LiveView as well. Note the difference in the documentation of the attributes of for example form/1

https://hexdocs.pm/phoenix_live_view/0.18.3/Phoenix.Component.html#form/1 

screenshot_0_18_3_form_attributes


and

https://hexdocs.pm/phoenix_live_view/0.18.4/Phoenix.Component.html#form/1


screenshot_0_18_4_form_attributes

File: mix.exs [hunk 1]
```diff
@@ -27,7 +27,7 @@ defmodule HeroiconsElixir.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:phoenix_live_view, "~> 0.18.2"},
+      {:phoenix_live_view, "~> 0.19"},
```
Comment (mveytsman) [inline mix.exs]: Could we just do `>= 0.18.2` in order to make this not backwards incompatible?

Comment (dovadi) [reply mix.exs]: Yes we can of course, you solve the dependency lock issue on live view 0.18, so you're able to use live_view 0.19 as well ..... however it's not really a case of _functional_ backward incompatibility, so I prefer to stay in sync with the updates of live_view

Comment (mveytsman) [reply mix.exs]: I'm not sure I follow...

I'd like someone on live view 0.18 to be able to use newer versions of this library without upgrading live view.

Comment (dovadi) [reply mix.exs]: On second thought, I think your right, your solution is really practical

Comment (nathanmalishev) [reply mix.exs]: Added the suggested changes here - @mveytsman  @dovadi changed version back to the running 0.5.2 as well FYI

**Comment by mveytsman** [comment]
Thank you ❤️ ❤️ ❤️

**Pull request merged**